tests/modules/sightings/test_models.py

```diff
@@ -50,6 +50,10 @@ def test_sighting_create_and_add_encounters(db):
     logging.info(test_sighting.get_encounters())
     logging.info(test_encounter_a.get_sighting())
 
+    test_encounter_a.delete()
+    test_encounter_b.delete()
+    test_sighting.delete()
```

Comment:
Doesn't the deletion of the sighting delete the encounters?

Reply:
`test_sighting.delete_cascade()` would.  `Encounter.sighting_guid` is currently nullable=true ... but i wonder if we ever would want to leave encounters "orphaned" when deleting the sighting (alone, without cascading)?